restaurants/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db.models.base           import Model
+from django.db.models.deletion       import CASCADE, PROTECT
+from django.db.models.fields         import CharField, DateTimeField, DecimalField, URLField
+from django.db.models.fields.json    import JSONField
+from django.db.models.fields.related import ForeignKey
+
+class Menu(Model):
+    name                = CharField(max_length=45, unique=True)
+
+    class Meta():
+        db_table = "menus"
+
+class Category(Model):
+    menu                = ForeignKey(Menu, on_delete=PROTECT, related_name="category")
+    name                = CharField(max_length=45, unique=True)
+
+    class Meta():
+        db_table = "categories"
```

Comment:
`related name`을 아래와 같이 `one-to-many` 관계일 때, one에서 many를 역참조 해서 가져올 경우 사용되는 속성으로 복수형으로 만들어주셔야 합니다!

```suggestion
class Category(Model):
    menu = ForeignKey(Menu, on_delete=PROTECT, related_name="categories")
    name = CharField(max_length=45, unique=True)

    class Meta():
        db_table = "categories"
```

Reply:
복수형 완료했습니다!